### PR TITLE
[monodroid] Use CoreFX to implement System.Drawing types that used to be in Mono.Android.dll

### PR DESCRIPTION
**Pull request by alexischr**

### CODE REVIEW

**Comment by alexischr** [comment]
@monojenkins commit csproj and apidiff

File: mcs/class/Facades/System.Drawing.Common/TypeForwarders.cs [hunk 1]
```diff
@@ -10,6 +10,9 @@
 // System.Drawing.Primitives/ref:
 namespace System.Drawing
 {
+
+#if !MONODROID
```
Comment (marek-safar) [inline mcs/class/Facades/System.Drawing.Common/TypeForwarders.cs]: Why do you need the ifdef if the file is not used for MONODROID

Comment (alexischr) [reply mcs/class/Facades/System.Drawing.Common/TypeForwarders.cs]: Not sure I understand, this file is included. monodroid profiles sources include `System.Drawing.Common.dll.sources` which includes `TypeForwarders.cs`

Comment (marek-safar) [reply mcs/class/Facades/System.Drawing.Common/TypeForwarders.cs]: oh, I missed the #include

**Comment by alexischr** [comment]
@monojenkins backport to 2019-02

**Comment by akoeplinger** [comment]
Judging from the API Diff we're still missing a few `TypeForwardedFromAttribute("Mono.Android, ...")` as it's only on `SystemColors` right now.

Merging to get my other PR rolling but we need to keep track of this :)

**Comment by alexischr** [comment]
This commit https://github.com/mono/corefx/commit/68f67ad6ced8b74a65a1a5617a4c761bb203a25b was missing from corefx master so it will be picked up in the next bump. `2019-02` has it now too (via a new corefx `mono-2019-02` branch)

**Pull request merged**